_episodes/08-putting-it-all-together.md

```diff
@@ -7,8 +7,11 @@ questions:
     - "Why should I use Python to create plots?"
 objectives:
     - "Import the pyplot toolbox to create figures in Python."
+    - "Adapt maptlotlib objects created by Pandas or plotnine."
```

Comment:
> Adapt ... objects

might be not super clear. Perhaps, `Learn how to change / adjust plots... ` or something like that

Reply:
Edited to better reflect role of matplotlib